src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/UserController.kt

```diff
@@ -57,18 +58,36 @@ class UserController(
   ) = userDetailsService.get(tokenExtractor.getTokenFromHeader(bearerToken).userUuid()).toResponse()
 
   fun AddUserDetailsRequest.toUserDetails(userId: UserId) =
-    UserDetails(userId, firstName, lastName, signature, email, phoneNumber, OffsetDateTime.now(clock))
+    UserDetails(
+      userId,
+      firstName,
+      lastName,
+      signature,
+      email,
+      phoneNumber,
+      caseworkerBand,
+      OffsetDateTime.now(clock)
+    )
 }
 
 fun UserDetails.toResponse() =
-  UserDetailsResponse(this.userId(), this.firstName, this.lastName, this.signature, this.email, this.phoneNumber)
+  UserDetailsResponse(
+    this.userId(),
+    this.firstName,
+    this.lastName,
+    this.signature,
+    this.email,
+    this.phoneNumber,
+    this.caseworkerBand
+  )
 
 data class AddUserDetailsRequest(
   val firstName: FirstName,
   val lastName: LastName,
   val signature: String,
   val email: Email,
-  val phoneNumber: PhoneNumber
+  val phoneNumber: PhoneNumber,
+  val caseworkerBand: CaseworkerBand = CaseworkerBand.FOUR_PLUS // TODO remove defaulting once UI change released
```

Comment:
Would be nice to have a ticket Id in the // TODO

Reply:
Not moving the back end ticket to done until this is done.